### ISSUE TITLE
[BUG] nightly repos - version build numbers make automated installs of nightlys difficult

### ISSUE THREAD

**Issue by ITJamie** [saltstack/salt#64927]
**Description**
nightly repos do not have a sane version syntax that can be assumed,
adding a symlink from "minor/latest" to "minor/{build_version}" will fix the issue. otherwise scraping of the folder is needed for automated installs

**Additional context**
previous pr allowed us to get further down the nightly install chain (https://github.com/saltstack/salt/pull/64907)

**Comment by OrangeDog** [comment]
Is this part of #64917?

**Comment by ITJamie** [comment]
> Is this part of #64917?

no thats a separate issue

**Comment by s0undt3ch** [comment]
Bare with me.
We both agree that the current nightly, if you always want to latest nightly is cumbersome.

How about, taking https://repo.saltproject.io/salt-dev/3006.x/latest as the base URL:

```
salt/py3/debian/11/amd64/latest -> salt/py3/debian/11/amd64/minor/3006.1+298.g08482ed6f3/
```
```
salt/py3/redhat/8/x86_64/latest -> salt/py3/redhat/8/x86_64/minor/3006.1+298.g08482ed6f3/
```
Then `salt/py3/redhat/8/x86_64/latest.repo` is either a link to `salt/py3/redhat/8/x86_64/minor/nightly.repo` or a separate one, which would point to `salt/py3/redhat/8/x86_64/minor/<salt-version>` ...

Thoughts?

**Comment by ITJamie** [comment]
That works for me. But we need to be 100% certain that the links inside that latest.repo are actually pointing to the latest path instead of the builddate/version numbered paths. Since those files get downloaded only once (hence me making the pr with the extra file)
